CuratedContent/ExacutableEnvironments.md

```diff
@@ -0,0 +1,67 @@
+# Executable Environments for Software, Data, and Publication
+<!--deck text start-->
+There are several environments that support collecting data, software, and publication content in order to support generating a paper, reproducing its results, and exploring its parameter space. This article lists multiple resources, found below. 
+<!--deck text end-->
```

Comment:
Data collection is an important step for all fields of research and more importantly for published  scientific articles that are based on the research.  A very critical objective of data collection is to ensure the information and reliable data is collected so data-driven decisions can be made. The software products described are indicative of data collection methods for analyzing and publishing studies of outcomes of research and these quality data collection methods improve accuracy and validity for publications. Data collection is done before high quality research can begin so questions can be answered. 
Data collection is critically important for making informed  decisions from further analysis, study, and research. Data collection and analysis allows researchers to stay on top of trends, provide answers to problems, and analyze new insights to great effect.
Research papers are often based on huge amounts of summarized data that can easily read and understood. When writing and preparing to publish a research paper, it is important for data to be analyzed and presented in a visually appealing way. These environments describe some of the software available for collecting and publication.